src/components/MessageList.js

```diff
@@ -34,19 +36,17 @@ class MessageList extends PureComponent {
     this.notificationTimeouts = [];
   }
   static propTypes = {
-    /** The attachment component to render, defaults to Attachment */
-    Attachment: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
-    /** A list of immutable messages */
-    messages: PropTypes.array.isRequired,
-    /** Via Context: The channel object */
-    channel: PropTypes.object.isRequired,
-    /** Via Context: The function to update a message, handled by the Channel component */
-    updateMessage: PropTypes.func.isRequired,
-    /** Via Context: The function is called when the list scrolls */
-    listenToScroll: PropTypes.func,
-    /** Typing indicator component to render  */
+    /**
+     * Typing indicator UI component to render
```

Comment:
Can we do this for every UI component. Had someone ask about the Message prop on MessageList and it was missing from the docs. Saw you added that! Would be awesome if we can have: "xxx UI component to render" for all these props.